airgun/views/contenthost.py

```diff
@@ -29,6 +29,7 @@
 from airgun.widgets import EditableEntrySelect
 from airgun.widgets import ReadOnlyEntry
 from airgun.widgets import Search
+from airgun.widgets import Pagination
```

Comment:
Looks like `reorder_python_imports` is bothered by this line. I guess it would prefer to have it above `from airgun.widgets import ReadOnlyEntry`

Reply:
ACK, I ran reorder-python-imports views/contenthost.py widgets.py

Reply:
Travis does not like my change, I will revert it and try just what you suggested.

Reply:
strange, but that works